### ISSUE TITLE
Add more projection types

### ISSUE THREAD

**Issue by loreclem** [loreclem/areadef#6]
Add more projection types, in addition to `stere` and `eqc`.

**Comment by gerritholl** [comment]
EPSG codes would be nice to have.

**Comment by loreclem** [comment]
Good idea. At the moment the script is really dumb and a lot of things are hardcoded, but proj4.js supports EPSG codes, so it should be feasible. Which additional projections do you think are most urgent to add?

**Comment by gerritholl** [comment]
> Which additional projections do you think are most urgent to add?

I don't quite understand how this package works, so I don't know about the feasibility, but just a field where the user can enter any EPSG code?  Would that work?
